### DISCUSSION TITLE
Make ride construction next/previous section buttons holdable

### DISCUSSION THREAD

**Discussion by mrmagic2020** [OpenRCT2/OpenRCT2#22171] [Feature requests] [answered]
Currently, to navigate through a tracked ride during construction, one needs to repeatedly click on the next/previous section button, which is a pain when constructing long non-circuit rides or trying to close the circuit. It is possible to assign a shortcut key to the actions, but this may not be user-friendly for touch users.

In PR #21950, ride construction and demolish buttons are made holdable. I propose that we do the same to the next/previous section buttons.

Demonstration of the proposed change:

https://github.com/OpenRCT2/OpenRCT2/assets/104749673/88a347b2-42ae-43a1-8066-4e4d0adc16d7

**Comment by Gymnasiast** [comment]
Seems fine to me, could you make a PR?

**Reply by mrmagic2020** [reply, marked as answer]
Done - #22196 :)
